matplotlib_scalebar/scalebar.py

```diff
@@ -110,12 +110,13 @@ class ScaleBar(Artist):
                   'center':       10,
               }
 
-    def __init__(self, dx, units='m', dimension=SI_LENGTH, label=None,
+    def __init__(self, dx, units='m', dimension='si-length', label=None,
                  length_fraction=None, height_fraction=None,
                  location=None, pad=None, border_pad=None, sep=None,
                  frameon=None, color=None, box_color=None, box_alpha=None,
                  scale_loc=None, label_loc=None, font_properties=None,
-                 label_formatter=None, fixed_value=None, fixed_units=None):
+                 label_formatter=None, fixed_value=None, fixed_units=None,
+                 use_blit=False):
```

Comment:
Wouldn't `animated` be a better name for the argument? This is what matplotlib's `Artist` is using.

Reply:
Done!